### ISSUE TITLE
What's the shape of the input param actions in logprobs_and_entropy in distributions.py?

### ISSUE THREAD

**Issue by BoyuanYan** [ikostrikov/pytorch-a2c-ppo-acktr#56]
Firstly, I can't run this program cause I was in trouble while installing mujoco-py in newest miniconda environment. So, it's hard for me to insert print() or other methods to check the program.
However, there is a question about the input parameter named **actions** in [logprobs_and_entropy](https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/blob/master/distributions.py#L29). Suppose the shape of the other input parameter named **x** is (N, M), then shape of **actions** should be (N,1) according to 
`action_log_probs = log_probs.gather(1, actions)`. But in [main.py#L154](https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/blob/master/main.py#L154), **actions**'s shape is (N,action_shape) according to `rollouts.actions.view(-1, action_shape)`.
Could you please explain it ? Thanks very much!

**Comment by ikostrikov** [comment]
Hi,

you can use a free alternative to mujoco: bullet.

In main it handles both categorical and gaussian distributions while in logprobs_and_entropy it handles only categorical.

**Comment by BoyuanYan** [comment]
Thanks. I know where I am wrong. I thought action_shape is size of action space, not action dimension.
Btw, I just copy `baselines` on root directory to enable this program. 
And another question, there is no learning rate adjustment, which is weird for me. Why you choose a very little lr, and make no changes on it?

**Comment by ikostrikov** [comment]
It's challenging to pick good hyper parameters for RL algorithms. For this reason, I simply took parameters offered by OpenAI. Since they have more resources to perform hyper parameter search.

**Comment by BoyuanYan** [comment]
Thanks for your reply!!!
